tests/next_tests/integration_tests/feature_tests/iterator_tests/test_program.py

```diff
@@ -40,18 +39,10 @@ def copy_program(inp, out, size):
     )
 
 
+@pytest.mark.requires_gtir_program
 def test_prog(program_processor):
     program_processor, validate = program_processor
 
-    if program_processor in [
-        gtfn.run_gtfn.executor,
-        gtfn.run_gtfn_imperative.executor,
-        gtfn.run_gtfn_with_temporaries.executor,
-        dace.run_dace_cpu.executor,
-    ]:
-        # TODO(havogt): Remove, skipped during refactoring to GTIR
```

Comment:
maybe put the comment on the marker in definitions, because after refactoring to gtir the marker should be removed